app.py

```diff
@@ -3786,6 +3786,50 @@ def stop_machine2(machine_id, audit_id=None):
     threading.Thread(target=target, args=(machine.id, audit.id)).start()
 
 
+@app.route("/unshare_machine_from_self", methods=["POST"])
+@limiter.limit("100 per day, 10 per minute, 1/3 seconds")
```

Comment:
Since this is a very lightweight action the rate-limiting can be usual "60 per minute"